portal-api/api-tests/v1/utilisation-report-service/previous-reports.api-test.js

```diff
@@ -84,30 +97,9 @@ describe('GET /v1/previous-reports/:bankId', () => {
     expect(status).toEqual(401);
   });
 
-  it('returns the requested resource', async () => {
-    const expectedResponse = [
-      {
-        year: 2023,
-        reports: [
-          {
-            month: 'January',
-            path: 'www.abc.com',
-          },
-          {
-            month: 'February',
-            path: 'www.abc.com',
-          },
-          {
-            month: 'March',
-            path: 'www.abc.com',
-          },
-        ],
-      },
-    ];
-
-    const response = await as(aPaymentReportOfficer).get(previousReportsUrl(matchingBankId));
+  it('returns a 200 response for a valid request', async () => {
+    const { status } = await as(aPaymentReportOfficer).get(previousReportsUrl(matchingBankId));
 
-    expect(response.status).toEqual(200);
-    expect(JSON.parse(response.text)).toEqual(expectedResponse);
+    expect(status).toEqual(200);
```

Comment:
Are we sure about removing the check of the return body type? If the date is that big of a pain maybe only check the month, year and azureFileInfo?

Reply:
I'm pretty sure. Happy to wait for @ChristianMcc to approve instead though if you're uncomfortable with this.

Reply:
Found a way to assert on the response without having to manually map all the `ObjectId` (for mongo `_id` values) and `Date` objects! Maybe there is some value in a single happy path test to at least make sure the correct controller is called. Either way I think it would still be interesting to have this discussion.

Reply:
lol I love the hack of stringifying and then posing as JSON dad

Reply:
I think I would have been fine with just checking status and treating this as just a green/red on can we get a good response. The stringify/parse hack is nicer though for sure and a nicer form of regression testing so works for me :)